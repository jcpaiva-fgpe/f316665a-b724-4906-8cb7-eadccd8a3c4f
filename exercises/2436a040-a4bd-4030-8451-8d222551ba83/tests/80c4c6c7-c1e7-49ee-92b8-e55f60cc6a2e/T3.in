1 1 1 0 0 0 1 0 0 1 0 0 0 0 1 1 1 0 0 0
0 1 0 0 0 0 0 1 0 0 0 0 0 0 0 1 0 0 0 0
0 1 0 0 0 0 1 0 0 1 1 0 1 0 0 1 0 0 0 0
0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 1 0
1 0 1 0 1 0 1 0 0 0 0 0 1 0 0 0 0 0 1 0
0 1 0 0 0 1 0 0 0 0 0 1 0 1 0 0 0 1 1 1
0 0 0 0 0 0 0 1 0 0 0 0 0 0 0 1 0 0 0 0
0 0 0 0 0 0 0 1 1 1 0 0 0 0 1 0 0 0 0 0
0 1 1 0 0 0 0 1 0 0 1 0 1 0 0 1 0 1 0 0
0 0 0 0 0 0 0 0 0 0 0 1 0 0 0 0 0 1 0 0
8
1 5 3 5 6 9
1 4 4 4 10 3
1 4 2 5 7 10
2 4 4 4 4 3
5 1 5 5 4 3
1 5 2 10 3 4
5 5 2 10 3 4
4 6 5 6 6 6
