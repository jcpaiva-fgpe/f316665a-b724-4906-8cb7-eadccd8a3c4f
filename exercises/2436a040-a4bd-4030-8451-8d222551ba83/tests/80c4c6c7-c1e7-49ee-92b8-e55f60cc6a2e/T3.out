Jogador 1: P2
Jogador 2: Agua
Jogador 1: Agua
Jogador 2: Invalida
Jogador 2: H2
Jogador 1: Invalida
Jogador 1: Agua
Jogador 2: H1
